### PR TITLE
Fix spinner styles

### LINKED ISSUE #1666: MvxAppCompatSpinner's Adapter's SimpleViewLayoutId should probably be Android.Resource.Layout.SimpleSpinnerItem
This needs more investigation but I THINK it's correct.  At least it looks correct on one of my phones.

See: https://github.com/android/platform_frameworks_support/blob/d6d7055c3064a711ef5311b23f57f752a2560a79/v7/appcompat/src/android/support/v7/widget/AppCompatSpinner.java#L268-L274

### PR DESCRIPTION
**Pull request by kjeremy**
Use the appropriate attributes to match platform and appcompat defaults
Fixes #1666

### CODE REVIEW

File: MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj [hunk 1]
```diff
@@ -138,6 +139,9 @@
       <Name>MvvmCross.Droid.Support.Core.UI</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\Xml\Empty.xml" />
```
Comment (martijn00) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj]: What's the empty xml for?

Comment (kjeremy) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj]: Without that file (or an empty `strings.xml` etc) Xamarin will not make `Resource.Layout.support_simple_spinner_dropdown_item` accessible.

You can't simply reference: `Android.Support.v7.AppCompat.Resource.Layout.support_simple_spinner_dropdown_item` in your code.  Instead Xamarin pulls those resources into your app's namespace during the build but only if you trick it into generating resources.

Comment (martijn00) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj]: I thought you could enable generating of the resources from the csproj file?

Comment (kjeremy) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj]: How?  It was already generating the `Resource.Designer.cs` file but until I added that `Empty.xml` it was 0 kb.

Comment (martijn00) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj]: Looking at our sample project, this one misses some lines. But i have no idea if it actually works like that: https://github.com/MvvmCross/MvvmCross/blob/develop/TestProjects/Android-Support/Fragments/Example.Droid/Example.Droid.csproj#L13

Maybe this is even a bug.

Comment (kjeremy) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj]: Hm.  I'll try that out.  For reference I did file a bug (see the second commit message).

Comment (kjeremy) [inline MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj]: Adding `AndroidResgenClass` didn't help.

**Comment by sisterray** [comment]
HI Is there a working example for this. I am getting an exception when touching the spinner to show the list. This worked in 4.2.3 I am in the process of upgrading to v.5.4.2.

The problem is outlined in [MvxAppCompatSpinner Crash](https://stackoverflow.com/questions/47330202/after-upgrading-to-mvvmcross-v5-4-2-mvxappcompatspinner-crashes-app-on-attemptin)

**Pull request merged**